subprojects/configuration-cache/configuration-cache.gradle.kts

```diff
@@ -83,6 +83,9 @@ dependencies {
     integTestImplementation(libs.guava)
     integTestImplementation(libs.ant)
     integTestImplementation(libs.inject)
+    integTestImplementation(libs.jettyWebApp) {
+        because("A bug in the Groovy compiler leaks API dependencies")
+    }
```

Comment:
I think it would make sense to link to the bug here, so we can remove this once it's not needed.

Reply:
It's supposed to be fixed: https://issues.apache.org/jira/browse/GROOVY-9352
I'll need to double check if _we_ leak or not. But it's something that would benefit from linting...

Reply:
So we'll remove this before the PR is merged? I think that's OK. Otherwise we should add the link to the issue in a comment.

Reply:
Yes, see latest commit.

Reply:
Thanks!